xinference/model/llm/llm_family.json

```diff
@@ -494,6 +494,15 @@
     ],
     "model_description": "ChatGLM3 is the third generation of ChatGLM, still open-source and trained on Chinese and English data.",
     "model_specs": [
+      {
+        "model_format": "ggmlv3",
+        "model_size_in_billions": 6,
+        "quantizations": [
+          "q4_0"
+        ],
+        "model_id": "Xorbits/chatglm2-6B-GGML",
+        "model_file_name_template": "chatglm2-ggml-{quantization}.bin"
```

Comment:
Additionally, I noticed that this model is not listed on Hugging Face. Should it not be available on both Hugging Face and ModelScope?

Reply:
Done.